kimchi/src/circuits/lookup/constraints.rs

```diff
@@ -24,7 +24,7 @@ use super::runtime_tables;
 pub const CONSTRAINTS: u32 = 7;
 
 /// The number of random values to append to columns for zero-knowledge.
-pub const ZK_ROWS: usize = 3;
+pub const EVALS: usize = 2;
```

Comment:
So the weird thing is that in `polynomial/permutation.rs` we already have

```rust
pub const ZK_ROWS: u64 = 3;
```

IMO all of these should live either in `kimchi/src/lib.rs`, or in some constant file `kimchi/src/constants.rs`

I think we could also have two constants:

```rust
/// The number of points that are evaluated in the protocol (zeta and zeta * omega).
pub const EVALS: u64 = 2;

/// The number of rows that are added to a witness/circuit in order to provide zero-knowledge. We might actually only need `EVALS` but let's be double safe.
pub const ZK_ROWS: u64 = EVALS + 1;
```

(also `u64` makes more sense here than `usize`)

wdyt? It looks less awkward than having `EVALS + 1` everywhere

Reply:
Yes, I agree with your idea.
Just one question, should we move the constants to `lib.rs` or `constants.rs`?
I think it is more meaningful to keep the constants as is.

Reply:
Let's say `constants.rs`. I don't think this is the current file makes sense : o

Reply:
Done https://github.com/o1-labs/proof-systems/pull/974/commits/91350343175bf874d16ada51a0ee486a6135db73